### PR TITLE
Add DSDrive by DidiSoft

### PR DESCRIPTION
**Pull request by eugeneloza**

### CODE REVIEW

**Comment by michaliskambi** [comment]
Place the image in htdocs/images/original_size . Generate all "derivatives" (most in WebP format now) using `make` in `htdocs/images`.  

But before that, call `make do-not-regenerate-existing-images` --- this way subsequent `make` will only create new images. You can assume images in repo are already OK. So in total do

```
cd htdocs/images/
make do-not-regenerate-existing-images
make
```

Refer to the images from PHP (like `gallery_link`) using the original extension, in `htdocs/images/original_size`, so most likely PNG.

**Comment by eugeneloza** [comment]
Also a question about format. I see that other images in the gallery refer to `*.png` but most of the physical images I've seen are `*.webp` : I should use png for original size image, right?

**Comment by michaliskambi** [comment]
> Also a question about format. I see that other images in the gallery refer to *.png but most of the physical images I've seen are *.webp : I should use png for original size image, right?

Images in `original_size` subdir should be `.png` for best quality (lossless compression) and widest support by all software. `.webp` can also have a lossless compression, but as it's not supported by ~100% of image-viewing/editing software, we prefer to keep originals as PNG.

The `.webp` is the format we generate to, right now. In general, the best practice for WEBP was to provide them only to browsers that support them (this can be known from HTTP headers), some things (Wordpress, Cloudflare) can also generate and provide WEBP transparently to browsers that support them. But in case of CGE static part of  website (our AsciiDoctor and PHP pages) I decided that it's simplest to just always server WEBP. It is 2023, all WWW browsers practically support  `.webp`.

Bottom line:
- commit .png to images/original_size/
- `make -C images` will generate thumbnails in WEBP format in various images/*/ subdirs
- don't worry about which format is made by `make -C images`. Just refer to image using original extension, so usually .png, when referring to it from our PHP function or AsciiDoctor macros.

**Review by michaliskambi — APPROVED** [review]
Looks good, thank you!

**Comment by michaliskambi** [comment]
@eugeneloza I updated CGE webpage to show on https://castle-engine.io/gallery_games.php result of all recent PRs. Thank you again!
